src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,19 @@
+package seedu.addressbook.data.tag;
+
+/**
+ * Association Class for Tag
+ */
+public class Tagging {
+    
+    public Tag tag;
+    public String action;
```

Comment:
think why `String` may not be the best data type for this particular field?
what would be more appropriate?